VideoLocker/res/layout/fragment_course_combined_info.xml

```diff
@@ -171,19 +172,9 @@
                         tools:targetApi="17" />
 
                     <Switch
+                        style="@style/edX.Widget.Switch"
```

Comment:
Hmmm. I tried both `android:switchStyle` and even just plain `switchStyle` and neither seemed to work.